trace-to-events/src/main.rs

```diff
@@ -65,6 +70,7 @@ async fn main() {
     );
 
     let producer: FutureProducer = client_config
+        .set("linger.ms", "0")
```

Comment:
Pretty sure the default value for `linger.ms` is 0ms anyway, is there a reason for setting this?